src/conn/conn_log.c

```diff
@@ -79,28 +79,23 @@ __logmgr_force_archive(WT_SESSION_IMPL *session, uint32_t lognum)
 }
 
 /*
- * __logmgr_set_majmin --
- *     Set the required major or minor of the given field. Wrapper for setting the required minimum
- *     and maximum fields in the connection.
+ * __logmgr_set_log_version --
+ *     Set the log version required for the given WiredTiger version.
  */
 static void
-__logmgr_set_majmin(uint16_t req_major, uint16_t req_minor, uint16_t *log_req)
+__logmgr_set_log_version(WT_VERSION version, uint16_t *log_req)
 {
-    /*
```

Comment:
There is a slight restructuring of the logic here, but behaviour is unchanged. 
For reference the `WT_LOG_VERSIONS` are:
```
#define WT_LOG_V2_VERSION ((WT_VERSION){3, 0, 0})
#define WT_LOG_V3_VERSION ((WT_VERSION){3, 1, 0})
#define WT_LOG_V4_VERSION ((WT_VERSION){3, 3, 0})
#define WT_LOG_V5_VERSION ((WT_VERSION){10, 0, 0})
```